androidApp/src/main/AndroidManifest.xml

```diff
@@ -73,8 +73,8 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
-<!--
+
```

Comment:
Should these be enabled in main?

Reply:
Seems like I forgot to comment those back after testing Auto.